app/gateway/upgrade/rolling.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Rolling upgrade for {{site.base_gateway}}"
+description: Learn how to perform a rolling upgrade for {{site.base_gateway}}.
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+
+tags:
+    - upgrades
+    - versioning
+
+related_resources:
+  - text: "Upgrading {{site.base_gateway}}"
+    url: /gateway/upgrade/
+  - text: "Dual-cluster upgrade"
+    url: /gateway/upgrade/dual-cluster/
+  - text: In-place upgrade
+    url: /gateway/upgrade/in-place/
+  - text: "{{site.base_gateway}} breaking changes"
+    url: /gateway/breaking-changes/
+  - text: "Backing up and restoring {{site.base_gateway}}"
+    url: /gateway/upgrade/backup-and-restore/
+  - text: "{{site.base_gateway}} CLI reference"
+    url: /gateway/cli/
+---
+
+The rolling upgrade strategy is a {{site.base_gateway}} upgrade option specifically designed 
+for [DB-less mode](/gateway/db-less-mode/), data planes running in [hybrid mode](/gateway/hybrid-mode/), and {{site.konnect_short_name}}.
+This strategy is meant for nodes that don't use a database and are independent of each other.
+
+This guide refers to the old version as cluster X and the new version as cluster Y.
+
+The rolling upgrade is a process of continuously adding new nodes of version Y, while shutting 
+down nodes of version X.
+
+{% mermaid %}
+flowchart TD
+    yml[fa:fa-file kong.yml]
+    CPX(Current 
+    Node X)
+    CPX2(Current 
+    Node X)
+    CPX3(Current 
+    Node X)
+    CPY(New 
+    Node Y)
+    CPY2(New 
+    Node Y)
+    CPY3(New 
+    Node Y)
+    LB(Load balancer)
+    API(API requests)
+
+    API --> LB & LB & LB & LB
+    LB -.-> CPX
+    LB -.90%.-> CPX2
+    LB -.-> CPX3
+    LB --> CPY
+    LB --10%--> CPY2
+    LB --> CPY3
+    CPX -.- yml
+    CPX2 -.- yml
+    CPX3 -.- yml
+    CPY -.- yml
+    CPY2 -.- yml
+    CPY3 -.- yml
+
+    style API stroke:none
+    style CPX stroke-dasharray:3
+    style CPX2 stroke-dasharray:3
+    style CPX3 stroke-dasharray:3
+    linkStyle 3,7,8,9,13,14,15 stroke:#b6d7a8
```

Comment:
```suggestion
    style API stroke:none!important
    style CPX stroke-dasharray:3
    style CPX2 stroke-dasharray:3
    style CPX3 stroke-dasharray:3
    linkStyle 3,7,8,9,13,14,15 stroke:#b6d7a8!important
```
This will force the styling to appear.